policyengine_us/parameters/gov/states/ri/tax/income/credits/eitc/match.yaml

```diff
@@ -0,0 +1,11 @@
+description: Rhode Island matches the federal EITC by this percentage.
+values:
+  2021-01-01: 0.15
+metadata:
+  unit: /1
+  label: Rhode Island EITC match percent
+  reference:
```

Comment:
could you add a legal reference?

Reply:
You didn't add a legal code reference, but I couldn't find one so I'll merge this and filed https://github.com/PolicyEngine/policyengine-us/issues/2715